src/qibosoq/programs/base.py

```diff
@@ -244,23 +239,23 @@ def acquire(
         # if there are no readouts, temporaray set 1 so that qick can execute properly
         reads_per_rep = 1 if readouts_per_experiment == 0 else readouts_per_experiment
 
-        res = super().acquire(
+        res = self.acquire(  # pylint: disable=E1123
             soc,
             readouts_per_experiment=reads_per_rep,
-            load_pulses=load_pulses,
-            progress=progress,
-            debug=debug,
+            load_pulses=True,
+            progress=False,
+            debug=False,
         )
         # if there are no actual readouts, return empty lists
         if readouts_per_experiment == 0:
             return [], []
         if average:
             # for sweeps res has 3 parameters, the first is not used
-            return res[-2:]
+            return np.array(res[-2]).tolist(), np.array(res[-1]).tolist()
```

Comment:
Are you sure you still need the double conversion?

This should be the result of a union (since `np.array` is normalizing both to an array, and then you turn it into a list).

Reply:
think I was getting some errors that could be explained with res[-2] and res[-1] being list of arrays... I didn't investigate much, but it got fixed like this

Reply:
Ok, this makes sense, I didn't consider. If they are lists of arrays, `np.array` is sufficiently powerful/flexible to convert them into arrays, and then `.tolist()` is converting them back into nested lists.

But are the lists of arrays coming from Qick?

Reply:
Yes, I was now looking at the qick source code and I am not sure to find the problem... I think that generally Qick returns arrays of arrays, but in some very specific cases does not follow the rule.
Again, I don't remember for which specific experiment I had to implement this fix, looking at the code it could be something relted to `acquire_decimated` that is used to acquire the non-averaged values

Reply:
Ok, let's classify this as a Qick issue. We could investigate later on (if ever).